<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Trajectory</class>
 <widget class="qMRMLWidget" name="Trajectory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>255</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="inputsCollapsibleButton">
     <property name="text">
      <string>Parameter</string>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="4">
       <widget class="QPushButton" name="TargetPointJumpSlicesButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Jump Target point to slices.</string>
        </property>
        <property name="text">
         <string>Target jump to slices</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="QPushButton" name="TargetPointButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Select Target Point.</string>
        </property>
        <property name="text">
         <string>Target Point</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2" colspan="3">
       <widget class="ctkSliderWidget" name="RadiusSliderWidget">
        <property name="toolTip">
         <string>Set path radius.</string>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
        <property name="pageStep">
         <double>2.000000000000000</double>
        </property>
        <property name="minimum">
         <double>0.500000000000000</double>
        </property>
        <property name="maximum">
         <double>50.000000000000000</double>
        </property>
        <property name="value">
         <double>2.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="EntryPointJumpSlicesButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Jump Entry point to slices.</string>
        </property>
        <property name="text">
         <string>Entry Jump to slices</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="CreateplanLabel">
        <property name="text">
         <string>Create plan</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="qMRMLNodeComboBox" name="Planselector">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Create or select a plan.</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLMarkupsFiducialNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="baseName">
         <string>Plan</string>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="addEnabled">
         <bool>true</bool>
        </property>
        <property name="removeEnabled">
         <bool>true</bool>
        </property>
        <property name="editEnabled">
         <bool>false</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
        <property name="selectNodeUponCreation">
         <bool>true</bool>
        </property>
        <property name="noneDisplay">
         <string>Select or create a Plan</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="ctkCheckBox" name="VisiblityButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Make model visible in slices.</string>
        </property>
        <property name="text">
         <string>Visibility in slice</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="RadiusLabel">
        <property name="text">
         <string>Radius(in mm)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QPushButton" name="EntryPointButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Generate curve.</string>
        </property>
        <property name="text">
         <string>Entry Point</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QCheckBox" name="VolumeReslicerButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="RadiusLabel_2">
        <property name="text">
         <string>Reslice Volume</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="LengthCollapsibleButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Length</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Length (in mm):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lengthLineEdit">
        <property name="text">
         <string>--</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCheckBox</class>
   <extends>QCheckBox</extends>
   <header>ctkCheckBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>inputsCollapsibleButton</tabstop>
  <tabstop>EntryPointButton</tabstop>
  <tabstop>EntryPointJumpSlicesButton</tabstop>
  <tabstop>TargetPointButton</tabstop>
  <tabstop>TargetPointJumpSlicesButton</tabstop>
  <tabstop>LengthCollapsibleButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>Trajectory</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>Planselector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
